quickstarts/microsoft.compute/glassfish-on-suse/azuredeploy.json

```diff
@@ -19,6 +19,7 @@
     "glassfishVersion": {
       "type": "int",
       "minValue": 3,
+      "maxValue": 4,
```

Comment:
remove minValue and maxValue property since there are allowedValues

Reply:
removed minValue and maxValue property